src/ngAnimate/module.js

```diff
@@ -267,9 +267,14 @@
  * .message.ng-enter-prepare {
  *   opacity: 0;
  * }
- *
  * ```
  *
+ * ### Animating any scope's variable change
```

Comment:
I think this title isn't very clear. It makes it sound as if emphasis is put on `any scope`.
AFAICT, what you are trying to say is "animating between arbitrary values" (i.e. not between two states but between any number of (unknown) states; similar to Angular's `* => *` (from any state to any state)).

<sub>Not sure what's a good heading though :flushed:</sub>

Reply:
Maybe something like:
- `Animating any value change of a directive`,
- `Animating between subsequent value changes`,
- or as you've written it `Animating between arbitrary values`.